devlab/Vagrantfile

```diff
@@ -1,39 +1,41 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby ts=2 sw=2 et sua= inex= :
 
-public_key_path = "#{ENV["HOME"]}/.ssh/id_rsa.pub"
+config_file = "config.ini"
+options = {}
+File.foreach(config_file) { |line|
+  option, value = line.split("=")
```

Comment:
You should only run these commands if the line is not empty, otherwise I get error:
```
$ vagrant status
There was an error loading a Vagrantfile. The file being loaded
and the error message are shown below. This is usually caused by
a syntax error.

Path: /Users/rverchikov/Documents/saic/CloudFerry/devlab/Vagrantfile
Message: undefined method `strip' for nil:NilClass
```